apis/schema/v1alpha1/mysqldatabase_webhook.go

```diff
@@ -95,42 +90,44 @@ func validateMySQLDatabaseUpdate(oldobj *MySQLDatabase, newobj *MySQLDatabase) e
 		return nil
 	}
```

Comment:
Don't comment out things. If not needed delete that.

Reply:
There are some logical dependancy. mysqldatabase_webhook.go file needs to be edited by tasdid.  I have just commented the error-containing part